Formula/fio.rb

```diff
@@ -3,7 +3,7 @@ class Fio < Formula
   homepage "https://github.com/axboe/fio"
   url "https://github.com/axboe/fio/archive/fio-3.19.tar.gz"
   sha256 "809963b1d023dbc9ac7065557af8129aee17b6895e0e8c5ca671b0b14285f404"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
At least some of their source appears to be `GPL-2.0-or-later`: https://github.com/axboe/fio/search?q=at+your+option

Reply:
Sure but because some of it is not (https://metadata.ftp-master.debian.org/changelogs//main/f/fio/fio_3.25-2_copyright ) I would recommend going with the most restrictive option.

Reply:
Then we can use

    license all_of: ["GPL-2.0-only", "GPL-2.0-or-later"]

Reply:
It's really is a mixture and you would have to include all the bits that the Debian packager pointed out if you go that route. I would strongly suggest following what the Linux distros do and listing it as GPL v2 only...

Reply:
You're right, let's include all that's there.

    license all_of: ["GPL-2.0-only", "GPL-2.0-or-later", "BSD-2-clause", :public_domain]